bin/git/hooks/pre-commit/validate-diffs

```diff
@@ -25,3 +26,12 @@ fi
 git diff --cached --name-only --diff-filter=A -z $against |
         LC_ALL=C tr -d '[ -~]\0' | wc -c
 check_rc "Rename non-ASCII file name(s) before committing"
+
+step_name 'Checking for trailing whitespace'
```

Comment:
Should you be a bit more specific here? `'Checking for trailing whitespace in YAML/MD files'`